src/app/components/currency/currency.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from '@testing-library/react';
+import Currency, { CurrencyProps } from './currency';
+
+describe(Currency.name, () => {
+  test('should format the given amount in INR currency format', () => {
+    const props: CurrencyProps = {
+      amount: 12345
+    };
+
+    render(<Currency {...props}></Currency>);
+
+    screen.getByText('₹12,345.00');
```

Comment:
probably want to use data test id to get elemnent and test for text?

Reply:
yup, but have seen this pattern too since get throws an error if there isn't exactly one matching the criteria.
Not a fan either of the pattern, will make this change